cmd/neofs-node/netmap.go

```diff
@@ -182,9 +184,26 @@ func initNetmapService(c *cfg) {
 	addNewEpochAsyncNotificationHandler(c, func(ev event.Event) {
 		e := ev.(netmapEvent.NewEpoch).EpochNumber()
 
-		ni, err := c.netmapLocalNodeState(e)
+		var (
+			ni  *netmapSDK.NodeInfo
+			err error
+		)
+
+		expBackoff := backoff.NewExponentialBackOff()
+		expBackoff.InitialInterval = 200 * time.Millisecond
+		expBackoff.MaxElapsedTime = 5 * time.Second
+
+		err = backoff.RetryNotify(
+			func() error {
+				ni, err = c.netmapLocalNodeState(e)
+				return err
+			},
+			expBackoff,
+			func(err error, d time.Duration) {
+				c.log.Warn("retrying due to error", zap.Uint64("epoch", e), zap.Error(err), zap.Duration("retry-after", d))
+			})
 		if err != nil {
-			c.log.Error("could not update node state on new epoch",
+			c.log.Error("could not update node state on new epoch after retries",
```

Comment:
is retry guaranteed to happen? Cuz if not, this wont be always true

we can count and write num of attempts instead